apps/smeargle/src/routes/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { createResource, Show } from "solid-js";
+import { isServer } from "solid-js/web";
+import { useRouteData } from "solid-start";
+import { Teaser as TeaserType } from "tonemato-types";
+import Teaser from "~/components/Teaser";
+
+async function fetchRalts() {
+  console.log("server: ", isServer);
+  const response = await fetch(
+    isServer ? "http://ralts:3005/" : "http://localhost:3005/"
+  );
+  return await response.json();
+}
+
+export function routeData() {
+  return createResource(fetchRalts);
+}
+
+export default function Home() {
+  const [data] = useRouteData<typeof routeData>();
+
+  const error = <div>something went wrong</div>;
+
+  return (
+    <main class="mx-auto  p-4 font-montserrat">
+      <p>Hello</p>
+      <Show when={!data.error} fallback={error}>
+        <div class="grid grid-cols-2 gap-6">
+          {data()?.cache?.backstagepro?.pages[0]?.map((teaser: TeaserType) => (
+            <Teaser teaser={teaser} />
+          ))}
+        </div>
+      </Show>
+    </main>
```

Comment:
The Code breaks on my machine by using `Show` but idk why (╯°□°）╯︵ ┻━┻, see Video attached:

https://user-images.githubusercontent.com/18482238/231606692-1f4687e7-33e4-4317-9d22-367a65e38f9e.mp4




```suggestion
    <main class="mx-auto  p-4 font-montserrat">
      <p>Hello</p>
      <Switch fallback={<div>Not Found</div>}>
        <Match when={data.state === 'pending' || data.state === 'unresolved'}>
          Loading...
        </Match>
        <Match when={data.state === 'ready'}>
          <div class="grid grid-cols-2 gap-6">
            {data().cache.backstagepro.pages[0].map((teaser: TeaserType) => (
              <Teaser teaser={teaser} />
            ))}
          </div>
        </Match>
        <Match when={data.state === 'errored'}>
          {JSON.stringify(data.error)}
        </Match>
      </Switch>
    </main>
```

Reply:
doesnt look like a fix to me although I kinda like the Switch pattern :D To me it looks like some kind of issue with the server side rendering/fetching. As soon as you switch to client side fetching (by routing to about and back) it seems to work just fine but the initial page load fetch seems to run into a problem (loading forever)... gonna look into it but would also love to see the network tab on this :[

Reply:
Yep seems to be an ssr issue... The client side fetch works just fine but the server side render is stuck in the loading state... strange... personally dont have that problem but gonna look into that - is the frontend running in a container or 'locally'? Cause it needs run in the docker stack to propperly work with ssr right now (client fetches from localhost and the server from docker internal network)

Reply:
Just started the project again and now it just works 😅. Was running it in containers yesterday and today. Maybe the build process was somehow stuck or something idk.

> (client fetches from localhost and the server from docker internal network) - maybe I should change that btw

I think that is actually the best we can do tbh :D.